README.md

```diff
@@ -1,66 +1,29 @@
-# Twilio Voice React Native App
+# Twilio Voice React Native Reference
 
-This is a React Native App that demonstrates [Twilio Voice React Native SDK](https://github.com/twilio/twilio-voice-react-native) usage.
+## What is this Project?
 
-## Prerequisites
+The Twilio Voice React Native Reference App is an example implementation of the Twilio Voice React Native SDK and serves to inspire developers who want to leverage the power of Twilio Programmable Voice in their React Native applications.
 
-- [React Native CLI](https://reactnative.dev/docs/environment-setup)
-- [Yarn](https://yarnpkg.com/getting-started)
-- Token - Create a file, `e2e-tests-token.ts` under `example/src` directory with the following content.
-  ```ts
-  export const token = 'my-token-goes-here';
-  ```
+## Before you use this Project
 
-## Setup
+You should have the React Native development environment set up and have a firm understanding of Twilio Programmable Voice.
 
-1. Clone this repository
-  ```sh
-  git clone https://github.com/twilio/twilio-voice-react-native.git
-  ```
-2. Switch to the root directory
-  ```sh
-  cd twilio-voice-react-native
-  ```
-3. Install SDK dependencies
-  ```sh
-  yarn install
-  ```
-4. Build the constants files for each platform
-  ```sh
-  yarn run build:constants
-  ```
-5. Switch to the example directory
-  ```sh
-  cd example
-  ```
-6. Install app dependencies
-  ```sh
-  yarn install
-  ```
-7. Install pods for iOS
-  ```sh
-  cd ios
-  pod install
-  ```
+Please follow the official React Native guides to set up your environment, and additionally the Twilio Programmable Voice documentation. TODO LINKS
 
-## Running the app
+### Structure
 
-Switch to the example directory
-```sh
-cd example
-```
+This repository is a monorepo hosting two major components.
 
-Start metro bundler:
-```sh
-yarn start
-```
+* A React Native application under:
+  ```
+  app/
+  ```
+  The example application uses the Twilio Voice React Native SDK and features a dialpad to make outgoing calls, and registration to receive incoming calls.
 
-On a different terminal window, run on Android:
-```sh
-yarn run android
-```
+* A complimentary backend server under:
```

Comment:
No need to mention complimentary

Reply:
Ack, I was thinking of the mathematical definition of `complement` but it could be weird if it's construed the other way.